### DISCUSSION TITLE
Welcome to vault-plugin-auth-u2f Discussions!

### DISCUSSION THREAD

**Discussion by bruj0** [bruj0/vault-plugin-auth-u2f#4] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mohammad228** [comment]
Hello
I have a Yubi HSM and a Hashicorp vault community edition. could I use this plugin for store seal and unseal key to yubi HSM ( i want to use that as a Yubi key)?
